config/pipelines/high_throughput_scrna_core_cdna_prep.yml

```diff
@@ -23,6 +23,11 @@ scRNA Core cDNA Prep GEM-X 5p GEMs Input:
     request_type_key: limber_scrna_core_cdna_prep_gem_x_5p_gems_input
   relationships:
     LRC GEM-X 5p GEMs Input: LRC GEM-X 5p cDNA PCR
+    LRC GEM-X 5p cDNA PCR: LRC GEM-X 5p cDNA PCR XP
+scRNA Core cDNA Prep GEM-X 5p GEMs Input CITE:
+  pipeline_group: scRNA Core cDNA Prep
+  filters:
+    request_type_key: limber_scrna_core_cdna_prep_gem_x_5p_gems_input
```

Comment:
should this be the _cite request type? or is it the same for this single step?

Reply:
think it's fine

Reply:
Yes, it is the same for this step